popstats/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "hemi-pop-stats",
+  "version": "1.0.0",
+  "description": "Report PoP payouts to Absinthe",
+  "license": "MIT",
+  "author": {
+    "name": "Gabriel Montes"
```

Other packages in this repository use the author "Hemi Labs". I think we should do the same here for consistency.
```suggestion
    "name": "Hemi Labs"
```

---

I don't see a problem with it and prefer proper attribution. Unless it is a policy, let's keep it as is.